src/css/_field-cell.scss

```diff
@@ -15,7 +15,8 @@
     }
   }
 
-  &._path {
+  &._path,
+  &.event_type {
```

Comment:
This is how we give the event_type field a red background in the various places in the app.